db/db_impl_write.cc

```diff
@@ -173,6 +170,7 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
       }
     }
 
+    uint64_t last_sequence = versions_->LastSequence();
```

Comment:
Is all the tests pass after this change? (Don't know how to read the phabricator result after the fbcode migjration..)